### PR TITLE
Add blog post: Bringing bootc to AlmaLinux

### PR DESCRIPTION
**Pull request by imbev**

### CODE REVIEW

File: content/blog/2024-08-08-bootc-almalinux-heliumos.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ image: /users/imbev.jpg
+date: 2024-08-08
+images:
+  - /blog-images/2024/2024-08-08-bootc-almalinux.png
+post: 
+    title: "Bringing bootc to AlmaLinux"
+    image: /blog-images/2024/2024-08-08-bootc-almalinux.png
+---
+
+bootc is a new deployment method that powers [Image mode for RHEL](https://www.redhat.com/en/about/press-releases/red-hat-reimagines-enterprise-linux-ai-future-image-mode-red-hat-enterprise-linux) and [HeliumOS](https://www.heliumos.org/), a new Linux distribution. We are excited to announce that AlmaLinux now offers [bootc images for Intel/AMD(x86_64)](https://quay.io/repository/almalinuxorg/almalinux-bootc?tab=tags).
```
Comment (codyro) [inline content/blog/2024-08-08-bootc-almalinux-heliumos.md:16]: What is the relevance of HeliumOS here besides it uses bootc? This doesn't seem appropriate here.

File: content/blog/2024-08-08-bootc-almalinux-heliumos.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## How did this start?
+My goal as developer of HeliumOS is to produce a Linux-based desktop operating system that is user-friendly, stable, and reliable. The first foray towards this goal used a Debian base, however this had some issues:
+
+- Debian has 3-5 years of support. Though stable relative to most Linux distributions, I wanted more stability.
+- An unexpected interruption during a traditional Linux package manager update can render the system un-bootable or otherwise broken.
+
+2 months ago, I learned of Image mode for RHEL, a new method of deploying RHEL that used OCI images for updates. Rebasing on Enterprise Linux and using bootc, the technology behind Image mode for RHEL would solve both of the issues that I found with Debian. After various experiments, HeliumOS [announced](https://www.heliumos.org/blog/post/heliumos-v9-alpha-is-available-for-downlaod/) an Alpha release based on CentOS Stream.
```
Comment (codyro) [inline content/blog/2024-08-08-bootc-almalinux-heliumos.md:24]: Seems odd to advertise a CentOS Stream derivative on the AlmaLinux blog.

Comment (imbev) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:24]: The CentOS Stream derivative is now an AlmaLinux derivative.

File: content/blog/2024-08-08-bootc-almalinux-heliumos.md (hunk 1 above)
Comment (imbev) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:16]: bootc support in AlmaLinux was largely motivated by HeliumOS, and there are very few projects currently using bootc.

More context in https://chat.almalinux.org/almalinux/channels/sigvirtcontainer

Comment (imbev) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:16]: If necessary, this post can be published on the HeliumOS blog instead and the AlmaLinux version of the post rewritten to be more AlmaLinux-focused.

Comment (codyro) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:16]: I've followed your bootc work in MM/GH, thank you so much for getting that moving for us! 

Could you make it more clear that AlmaLinux is now upstream to your project/HeliumOS? It'd make it a bit more clear how it's tied into bootc/Alma.

File: content/blog/2024-08-08-bootc-almalinux-heliumos.md (hunk 2 above)
Comment (codyro) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:24]: Marking resolve in lieu of the other thread.

File: content/blog/2024-08-08-bootc-almalinux-heliumos.md (hunk 1 above)
Comment (imbev) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:16]: Sure! I'll also add a starting point for other projects to use the AlmaLinux bootc images.

Comment (imbev) [reply content/blog/2024-08-08-bootc-almalinux-heliumos.md:16]: @codyro Is there anything else that needs to be changed?

**Review by codyro — APPROVED** [review]

**Comment by codyro** [comment]
Thanks for clarifying the relationship between the bootc images <-> HeliumOS <-> AlmaLinux :).

**Comment by imbev** [comment]
> Thanks for clarifying the relationship between the bootc images <-> HeliumOS <-> AlmaLinux :).

You're welcome :)

**Review by bennyvasquez — APPROVED** [review]
We'll merge this in Tuesday morning!

**Review by codyro — APPROVED** [review]

**Pull request merged**